src/components/AnnotationsPane.jsx

```diff
@@ -13,14 +13,15 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Utility } from '../lib/Utility';
 import { connect } from 'react-redux';
-import { constructLines, constructPoints, constructText } from '../lib/construct-line';
 
 class AnnotationsPane extends React.Component {
   constructor(props) {
     super(props);
     this.renderAnnotationInProgress = this.renderAnnotationInProgress.bind(this);
     this.renderAnnotations = this.renderAnnotations.bind(this);
     this.renderPreviousAnnotations = this.renderPreviousAnnotations.bind(this);
+    this.renderUserAnnotations = this.renderUserAnnotations.bind(this);
+    this.determineGreenLine = this.determineGreenLine.bind(this);
```

Comment:
[minor] At a glance, what a 'GreenLine' is isn't too clear, but to be fair, our logic's already pretty complex no matter what.

Reply:
Yeah, I spent a while trying to decide the best way to do this. With the streamlined `annotations` and `previousAnnotations`, it was a bit tricky to discern which annotation was selected and should display green. I'd be open to changing the variable name if you think that'd help readability, unless you have other suggestions.